src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/NavigationPage.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Page
+    x:Class="OfflineWorkflowSample.Views.NavigationPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource SystemControlAcrylicWindowBrush}">
+    <Grid>
+        <Grid.Resources>
+            <!-- This looks like it isn't being used, but it is. It overrides a style in NavigationView. -->
+            <!-- ReSharper disable once Xaml.RedundantResource -->
+            <SolidColorBrush x:Key="NavigationViewTopPaneBackground" Color="Transparent" />
+        </Grid.Resources>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <!-- Title bar -->
+        <Grid Background="Transparent"
+              HorizontalAlignment="Stretch"
+              Grid.Row="0">
+            <!-- Width of the padding columns is set in LayoutMetricsChanged handler. -->
+            <!-- Using padding columns instead of Margin ensures that the background
+                 paints the area under the caption control buttons (for transparent buttons). -->
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="60" />
+            </Grid.ColumnDefinitions>
+            <Image Source="../Assets/Square44x44Logo.png"
+                   Grid.Column="0"
+                   HorizontalAlignment="Left"
+                   VerticalAlignment="Center"
+                   Width="30" Height="30" Margin="4" />
+            <StackPanel x:Name="DraggablePart"
+                        HorizontalAlignment="Stretch"
+                        Background="Transparent"
+                        Grid.Column="1">
+                <TextBlock Text="{x:Bind ViewModel.Title,Mode=OneWay}"
+                           HorizontalTextAlignment="Center"
+                           Style="{StaticResource CaptionTextBlockStyle}"
+                           Margin="0,8,0,0" />
+            </StackPanel>
+        </Grid>
+        <!-- Everything else -->
+        <muxc:NavigationView x:Name="NavigationView"
+                             PaneDisplayMode="Top"
+                             Grid.Row="1"
+                             Background="Transparent"
+                             ItemInvoked="NavigationView_OnItemInvoked"
+                             IsBackEnabled="{x:Bind ContentFrame.CanGoBack,Mode=OneWay}"
+                             BackRequested="NavigationView_OnBackRequested">
+            <muxc:NavigationView.MenuItems>
```

Comment:
Whenever I use the "back" button in the UI, the highlighted item is not updated. This can be pretty confusing.